lighthouse-cli/performance-experiment/server.js

```diff
@@ -18,94 +18,62 @@
 'use strict';
 /**
  * @fileoverview Server script for Project Performance Experiment.
- * Start Server by calling startServer(port).
  *
  * Functionalities:
  *   Host report pages.
- *     Report pages should be stored in ./src/reports
- *     Report pages can be accessed via URL http://localhost:[PORT]/reports/[REPORT_FILENAME]
+ *     Report pages can be accessed via URL http://localhost:[PORT]/reports?key=[REPORT_KEY]
  */
 
 const http = require('http');
-const fs = require('fs');
 const parse = require('url').parse;
 const path = require('path');
+const opn = require('opn');
 const log = require('../../lighthouse-core/lib/log');
+const ReportGenerator = require('../../lighthouse-core/report/report-generator');
 
-const ROOT = `${__dirname}/src`;
-const FOLDERS = {
-  REPORTS: `${ROOT}/reports`
-};
-
-const server = http.createServer(requestHandler);
-server.on('error', err => log.error('PerformanceXServer', err.code, err));
+/**
+ * Start the server with an arbitrary port and open report page in the default browser.
+ * @param {!Object} lighthouseParams
+ * @param {!Object} results
+ * @return {!Promise<string>} Promise that resolves when server is closed
+ */
+let lhResults;
+function hostExperiment(lighthouseParams, results) {
```

Comment:
right now this seems more like `serveReport` or `serveAndOpenReport`.  Once experiments are in place, then I think we could rename whatever the entry method is called.